templates/nsq/0/rancher-compose.yml

```diff
@@ -0,0 +1,64 @@
+version: '2'
+catalog:
+  name: NSQD
+  version: v1.0.0-compat
+  questions:
+    - variable: "NSQD_SCALE"
+      description: "Number of NSQD nodes."
+      label: "Nsqd Nodes"
+      required: true
+      default: "3"
+      type: "int"
+    - variable: "NSQADMIN_SCALE"
+      description: "Number of NSQ Admin nodes."
+      label: "Nsqadmin Nodes"
+      required: true
+      default: "3"
+      type: "int"
+services:
+  nsqlookupd01:
+    scale: 1
+    start_on_create: true
+  nsqlookupd02:
+    scale: 1
+    start_on_create: true
+  nsqlookupd03:
+    scale: 1
+    start_on_create: true
```

Comment:
Why these 3 distinct `nsqlookupd0N` services if they are same?? Why not just one `nsqlookupd` scaled to 3 and use system discovery to run nsqd and nsqdadmin??

Reply:
the nsqlookupd servers are needed in `command:` as parameters for nsqd. they should be static and cant be scaled very well